app/views/products/_form.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= form_with model: @product, class: form_class do |f| %>
+
+  <p><%= intro_text %></p>
+
+  <%= f.label :name %>
+  <%= f.text_field :name %>
+
+  <%= f.label :merchant_id %>
+  <%= f.select :merchant_id, Merchant.where(id: session[:merchant_id]) %>
+  <%# need help displaying just merchant name  %>
+
```

Comment:
I need a little clarity here. I left a suggestion for getting merchant_id into params without displaying. Is the goal to display just the name?